chainerx_cc/chainerx/python/array.cc

```diff
@@ -139,9 +139,17 @@ py::object MakeCupyArrayFromArray(const py::module& m, py::handle self) {
 
 ArrayBodyPtr MakeArray(py::handle object, py::handle dtype, bool copy, py::handle device) {
     absl::optional<Dtype> dtype_ = dtype.is_none() ? absl::nullopt : absl::optional<Dtype>(GetDtype(dtype));
-    Device& dev = GetDevice(device);
 
-    return MakeArray(object, dtype_, copy, dev);
+    // TODO(take-cheeze): Add `GetDevice` that returns `absl::optiona<Device*>` to handle more strictly
```

Comment:
What do you mean by "more strictly"?

Reply:
There should be more places that shouldn't fallback to default device in python bindings.
I think C++ API is mostly safe since they handle optional device argument correctly.
Maybe I should place this comment to `python_internal::GetDevice`.

Reply:
Then, do you actually mean `absl::optional<std::reference_wrapper<Device>>`?
`Device*` is nullable by itself.

I can't yet understand what the word "strictly" means.
How about the following wording?

```suggestion
    // TODO(take-cheeze): Add `GetDevice` that returns `absl::optional<std::reference_wrapper<Device>>` and use it.
```